test/test.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"crypto/sha512"
+	"encoding/hex"
+	"fmt"
+	ledger "github.com/spacemeshos/go-ledger-sdk"
+	"github.com/spacemeshos/ed25519"
+)
+
+func uint64_to_buf(value uint64) []byte {
+	data := make([]byte, 8)
+	data[0] = byte((value >> 56) & 0xff)
+	data[1] = byte((value >> 48) & 0xff)
+	data[2] = byte((value >> 40) & 0xff)
+	data[3] = byte((value >> 32) & 0xff)
+	data[4] = byte((value >> 24) & 0xff)
+	data[5] = byte((value >> 16) & 0xff)
+	data[6] = byte((value >>  8) & 0xff)
+	data[7] = byte((value      ) & 0xff)
+	return data
+}
+
+func do_test(device *ledger.HidDevice) {
+	if err := device.Open(); err != nil {
+		return
+	}
+	defer device.Close()
+
+	version, err := device.GetVersion()
+	if err != nil {
+		fmt.Printf("get version ERROR: %v\n", err)
+	} else {
+		fmt.Printf("version: %+v\n", version)
+	}
+	publicKey, err := device.GetExtendedPublicKey(ledger.StringToPath("44'/540'/0'/0/0'"))
+	if err != nil {
+		fmt.Printf("get public key ERROR: %v\n", err)
+	} else {
+		fmt.Printf("public key: %+v\n", publicKey)
```

Comment:
This would be more useful (printing in hex):
```suggestion
		fmt.Printf("public key: %x\n", publicKey)
```